app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -332,6 +336,12 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
             }, "externalApp")
         }
 
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+            if (WebViewFeature.isFeatureSupported(WebViewFeature.FORCE_DARK_STRATEGY)) {
```

Comment:
Based on the documentation around `isFeatureSupported` I don't think you need the `SDK_INT` check before it, `isFeatureSupported` should handle it all.

Reply:
Ok I think the only part needed was the custom styles for v29 then :) I was beating my head around getting this to work properly so I tried lots of things lol

Reply:
In my testing it seems to continue to work as desired, so its cleaner now thanks for suggestion!